.env.example

```diff
@@ -4,7 +4,7 @@
 ENCRYPTION_KEY=6c1fe4e407b8911c104518103505b218
 
 # Required
-DB_CONNECTION_URI=postgres://infisical:infisical@db:5432/infisical
+DB_CONNECTION_URI=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}
```

Comment:
Are you sure env interpolation works this way with docker compose? I just did a quick search and i see: https://stackoverflow.com/questions/39699288/interpolate-variables-in-environment-variables-definition-with-docker-compose-ym

Reply:
https://docs.docker.com/compose/compose-file/12-interpolation/

Right in the documentation and tested in local